### ISSUE TITLE
Program with Default Arguments and Vectors Fails with NullPointerException 

### ISSUE THREAD

**Issue by 4e6** [enso-org/enso#1170]
### General Summary

Program produces `NullPointerException`

### Steps to Reproduce

```python
from Base import all

test_1 (x = 0) = [x]

test_2 = []

main = here.test_1
```

```
Execution finished with an error: java.lang.NullPointerException
        at <java> org.enso.interpreter.node.scope.ReadLocalVariableNode.readLong(ReadLocalVariableNode.java:47)
        at <java> org.enso.interpreter.node.scope.ReadLocalVariableNodeGen.executeAndSpecialize(ReadLocalVariableNodeGen.java:112)
        at <java> org.enso.interpreter.node.scope.ReadLocalVariableNodeGen.executeGeneric(ReadLocalVariableNodeGen.java:54)
        at <java> org.enso.interpreter.node.callable.ApplicationNode.executeGeneric(ApplicationNode.java:101)
        at <java> org.enso.interpreter.node.callable.function.BlockNode.executeGeneric(BlockNode.java:45)
        at <java> org.enso.interpreter.node.ClosureRootNode.execute(ClosureRootNode.java:69)
        at <enso> Main.test_1(Internal)
```

### Expected Result

Program finishes successfully.

### Actual Result

Program finishes with an error.

### Enso Version

main @ bd9689a2b43cceb2bab4f53be42a9796a6e7edfb

**Comment by iamrecursion** [comment]
This may be the issue that you've already found and fixed @kustosz.

**Comment by kustosz** [comment]
Most probably yes. Fix included in #1166

**Comment by kustosz** [comment]
Just tested, the issue is fixed there.
